lib/substitutions.json

```diff
@@ -2773,6 +2773,11 @@
     {
       "before": "stranger class",
       "after": "stranger-class"
+    },
+    {
+      "before": "real ly",
+      "after": "really",
+      "comment": "There's an empty em element in the middle of this word that gets converted to a space"
```

Comment:
Nice find with this one. I'll do a larger change to stop adding spaces in place of completely-empty elements; that fixes a lot of extra spaces (although most of those spaces are invisible).